src/Razor/src/Microsoft.AspNetCore.Razor.LanguageServer/Extensions/IServiceCollectionExtensions.cs

```diff
@@ -73,21 +73,11 @@ public static void AddFormattingServices(this IServiceCollection services)
         services.AddHandlerWithCapabilities<DocumentRangeFormattingEndpoint>();
     }
 
-    public static void AddCompletionServices(this IServiceCollection services, LanguageServerFeatureOptions featureOptions)
+    public static void AddCompletionServices(this IServiceCollection services)
```

Comment:
I look forward to you adding this parameter back in order to check the cohosting flag  😁